network-api/networkapi/templates/fragments/article_prev_next.html

```diff
@@ -0,0 +1,24 @@
+{% load i18n %}
+
+<div class="article-prev-next">
+  <div class="container">
+    <div class="row d-flex">
+      <div class="col-sm-6 text-right article-prev article-nav-arrow article-nav-arrow--left">
+        {% if page.prev_page %}
+          <small class="prev-next-text">{% trans 'Previous' %}</small>
+          <a href="{{ page.prev_page.url }}" class="article-prev-next-link">
+            {{ page.prev_page.title }}
+          </a>
+        {% endif %}
+      </div>
+      <div class="col-sm-6 article-next article-next-arrow article-nav-arrow article-nav-arrow--right">
+        {% if page.next_page %}
+          <small class="prev-next-text">{% trans 'Next' %}</small>
+          <a href="{{ page.next_page.url }}" class="article-prev-next-link">
```

Comment:
Playing with this, it feels like it makes more sense to have the entire region be the "button", rather than having a text link. That would also allow us to keep the text "normal text" without the underlying text decoration making things look a bit weird:

![image](https://user-images.githubusercontent.com/177243/94069787-01834500-fda6-11ea-8178-212131d0efd7.png)